### ISSUE TITLE
blosc/bloscpack dependency

### ISSUE THREAD

**Issue by thequackdaddy** [Blosc/bloscpack#46]
Hello,

I was trying to install bloscpack and noticed that `setup.py` has the following lines...

```
install_requires = [
    'blosc==1.2.7',
    'numpy',
    'six',
]
```

However, blosc appears to currently be at version 1.2.9dev0. Using pip/conda I wasn't able to install it either. 

Is it safe to just edit the requirements to be `blosc>=1.2.7`?

**Comment by esc** [comment]
You may try this at your own risk. Last time I tried a blosc > 1.2.7 many of the unit and or system tests were failing. However, these are very very picky at times since they check exact binary output and compression ratios -- so you may get away with using 1.2.9. It's hard to say for sure. 1.2.7 is likely to result in a fully passing test-suite. Let me know if 1.2.9 works for you.

On 29 March 2016 16:19:39 CEST, Peter Quackenbush <notifications@github.com> wrote:
>Hello,
>
>I was trying to install bloscpack and noticed that `setup.py` has the
>following lines...
>
>```
>install_requires = [
>    'blosc==1.2.7',
>    'numpy',
>    'six',
>]
>```
>
>However, blosc appears to currently be at version 1.2.9dev0. Using
>pip/conda I wasn't able to install it either. 
>
>Is it safe to just edit the requirements to be `blosc>=1.2.7`?
>
>---
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/Blosc/bloscpack/issues/46

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by thequackdaddy** [comment]
Right. I guess the problem is that I am unable to install blosc==1.2.7 on my Windows computer 

I have Anaconda 2.5 (Python 3.5.1) with Visual Studio community edition. I can compile and install 1.2.8 (or 1.2.9dev0 from github) with no problems. With 1.2.7, pip install blosc==1.2.7 gives me all this nastiness....

Both 1.2.8 and 1.2.9dev0 seem to work just fine. (I'm using these while playing with castra). 

```
uild\temp.win-amd64-3.5\Release\c-blosc/internal-complibs\zlib-1.2.8\inflate.obj
 build\temp.win-amd64-3.5\Release\c-blosc/internal-complibs\zlib-1.2.8\inftrees.
obj build\temp.win-amd64-3.5\Release\c-blosc/internal-complibs\zlib-1.2.8\trees.
obj build\temp.win-amd64-3.5\Release\c-blosc/internal-complibs\zlib-1.2.8\uncomp
r.obj build\temp.win-amd64-3.5\Release\c-blosc/internal-complibs\zlib-1.2.8\zuti
l.obj /OUT:build\lib.win-amd64-3.5\blosc\blosc_extension.cp35-win_amd64.pyd /IMP
LIB:build\temp.win-amd64-3.5\Release\blosc\blosc_extension.cp35-win_amd64.lib
    blosc_extension.obj : warning LNK4197: export 'PyInit_blosc_extension' speci
fied multiple times; using first specification
       Creating library build\temp.win-amd64-3.5\Release\blosc\blosc_extension.c
p35-win_amd64.lib and object build\temp.win-amd64-3.5\Release\blosc\blosc_extens
ion.cp35-win_amd64.exp
    shuffle.obj : error LNK2001: unresolved external symbol blosc_get_cpu_featur
es
    build\lib.win-amd64-3.5\blosc\blosc_extension.cp35-win_amd64.pyd : fatal err
or LNK1120: 1 unresolved externals
    error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\B
IN\\amd64\\link.exe' failed with exit status 1120

    ----------------------------------------
Command "C:\Anaconda3\python.exe -u -c "import setuptools, tokenize;__file__='C:
\\Users\\PQUACK~1\\AppData\\Local\\Temp\\pip-build-7u774njl\\blosc\\setup.py';ex
ec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\r\n', '\n'
), __file__, 'exec'))" install --record C:\Users\QUACK~1\AppData\Local\Temp\pip
-rbdprlu1-record\install-record.txt --single-version-externally-managed --compil
e" failed with error code 1 in C:\Users\QUACK~1\AppData\Local\Temp\pip-build-7u
774njl\blosc\

[Anaconda3] C:\Users\pquackenbush\git>
```

**Comment by esc** [comment]
Thanks for reporting. I suggest you open a ticket for this with the python-blosc project. As for installing bloscpack, feel free to clone the source code, remove the lines regarding the dependency and installing it. It should be fine if you just want to play around.

On 29 March 2016 23:01:49 CEST, Peter Quackenbush <notifications@github.com> wrote:
>Right. I guess the problem is that I am unable to install blosc==1.2.7
>on my Windows computer 
>
>I have Anaconda 2.5 (Python 3.5.1) with Visual Studio community
>edition. I can compile and install 1.2.8 (or 1.2.9dev0 from github)
>with no problems. With 1.2.7, pip install blosc==1.2.7 gives me all
>this nastiness....
>
>Both 1.2.8 and 1.2.9dev0 seem to work just fine. (I'm using these while
>playing with castra). 
>
>```
>uild\temp.win-amd64-3.5\Release\c-blosc/internal-complibs\zlib-1.2.8\inflate.obj
>build\temp.win-amd64-3.5\Release\c-blosc/internal-complibs\zlib-1.2.8\inftrees.
>obj
>build\temp.win-amd64-3.5\Release\c-blosc/internal-complibs\zlib-1.2.8\trees.
>obj
>build\temp.win-amd64-3.5\Release\c-blosc/internal-complibs\zlib-1.2.8\uncomp
>r.obj
>build\temp.win-amd64-3.5\Release\c-blosc/internal-complibs\zlib-1.2.8\zuti
>l.obj
>/OUT:build\lib.win-amd64-3.5\blosc\blosc_extension.cp35-win_amd64.pyd
>/IMP
>LIB:build\temp.win-amd64-3.5\Release\blosc\blosc_extension.cp35-win_amd64.lib
>blosc_extension.obj : warning LNK4197: export 'PyInit_blosc_extension'
>speci
>fied multiple times; using first specification
>Creating library
>build\temp.win-amd64-3.5\Release\blosc\blosc_extension.c
>p35-win_amd64.lib and object
>build\temp.win-amd64-3.5\Release\blosc\blosc_extens
>ion.cp35-win_amd64.exp
>shuffle.obj : error LNK2001: unresolved external symbol
>blosc_get_cpu_featur
>es
>build\lib.win-amd64-3.5\blosc\blosc_extension.cp35-win_amd64.pyd :
>fatal err
>or LNK1120: 1 unresolved externals
>error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio
>14.0\\VC\\B
>IN\\amd64\\link.exe' failed with exit status 1120
>
>    ----------------------------------------
>Command "C:\Anaconda3\python.exe -u -c "import setuptools,
>tokenize;__file__='C:
>\\Users\\PQUACK~1\\AppData\\Local\\Temp\\pip-build-7u774njl\\blosc\\setup.py';ex
>ec(compile(getattr(tokenize, 'open',
>open)(__file__).read().replace('\r\n', '\n'
>), __file__, 'exec'))" install --record
>C:\Users\QUACK~1\AppData\Local\Temp\pip
>-rbdprlu1-record\install-record.txt --single-version-externally-managed
>--compil
>e" failed with error code 1 in
>C:\Users\QUACK~1\AppData\Local\Temp\pip-build-7u
>774njl\blosc\
>
>[Anaconda3] C:\Users\pquackenbush\git>
>```
>
>
>---
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/Blosc/bloscpack/issues/46#issuecomment-203101977

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by esc** [comment]
@thequackdaddy will close this, feel free to re-open if you have further quetsions.

**Comment by thequackdaddy** [comment]
@esc Thank you. I forgot to open ticket with them. Thanks for reminder!

**Comment by esc** [comment]
FWIW this was resolved with bloscpack 0.11
